mullvad-relay-selector/src/relay_selector/mod.rs

```diff
@@ -5,46 +5,43 @@ mod helpers;
 mod matcher;
 mod parsed_relays;
 pub mod query;
+pub mod relays;
 
-use chrono::{DateTime, Local};
-use itertools::Itertools;
-use query::ObfuscationQuery;
-use std::{
-    path::Path,
-    sync::{Arc, LazyLock, Mutex},
-    time::SystemTime,
-};
+use matcher::{filter_matching_bridges, filter_matching_relay_list};
+use parsed_relays::ParsedRelays;
+use relays::{Multihop, Singlehop, WireguardConfig};
 
-use mullvad_types::{
-    constraints::Constraint,
-    custom_list::CustomListsSettings,
-    endpoint::MullvadWireguardEndpoint,
-    location::{Coordinates, Location},
-    relay_constraints::{
-        BridgeSettings, BridgeState, InternalBridgeConstraints, ObfuscationSettings,
-        OpenVpnConstraints, RelayConstraints, RelayOverride, RelaySettings, ResolvedBridgeSettings,
-        WireguardConstraints,
-    },
-    relay_list::{Relay, RelayEndpointData, RelayList},
-    settings::Settings,
-    wireguard::QuantumResistantState,
-    CustomTunnelEndpoint, Intersection,
-};
-use talpid_types::{
-    net::{
-        obfuscation::ObfuscatorConfig, proxy::CustomProxy, Endpoint, TransportProtocol, TunnelType,
-    },
-    ErrorExt,
+use crate::detailer::{openvpn_endpoint, wireguard_endpoint};
+use crate::error::{EndpointErrorDetails, Error};
+use crate::query::{
+    BridgeQuery, ObfuscationQuery, OpenVpnRelayQuery, RelayQuery, RelayQueryExt,
+    WireguardRelayQuery,
 };
 
-use crate::error::{EndpointErrorDetails, Error};
+use std::path::Path;
+use std::sync::{Arc, LazyLock, Mutex};
+use std::time::SystemTime;
+
+use chrono::{DateTime, Local};
+use itertools::Itertools;
 
-use self::{
-    detailer::{openvpn_endpoint, wireguard_endpoint},
-    matcher::{filter_matching_bridges, filter_matching_relay_list},
-    parsed_relays::ParsedRelays,
-    query::{BridgeQuery, OpenVpnRelayQuery, RelayQuery, WireguardRelayQuery},
+use mullvad_types::constraints::Constraint;
+use mullvad_types::custom_list::CustomListsSettings;
+use mullvad_types::endpoint::MullvadWireguardEndpoint;
+use mullvad_types::location::{Coordinates, Location};
+use mullvad_types::relay_constraints::{
+    BridgeSettings, BridgeState, InternalBridgeConstraints, ObfuscationSettings,
+    OpenVpnConstraints, RelayConstraints, RelayOverride, RelaySettings, ResolvedBridgeSettings,
+    WireguardConstraints,
 };
+use mullvad_types::relay_list::{Relay, RelayEndpointData, RelayList};
+use mullvad_types::settings::Settings;
+use mullvad_types::wireguard::QuantumResistantState;
+use mullvad_types::{CustomTunnelEndpoint, Intersection};
+use talpid_types::net::{
+    obfuscation::ObfuscatorConfig, proxy::CustomProxy, Endpoint, TransportProtocol, TunnelType,
+};
+use talpid_types::ErrorExt;
```

Comment:
Again, here nightly `rustfmt` make a ton of changes to the imports. I would prefer to trust its decision, but I'll leave it up to you.

Reply:
I don't really care, ran the nightly formatter